yt_dlp/extractor/bilibili.py

```diff
@@ -156,26 +146,28 @@ def _real_extract(self, url):
         mobj = self._match_valid_url(url)
         video_id = mobj.group('id_bv') or mobj.group('id')
 
-        av_id, bv_id = self._get_video_id_set(video_id, mobj.group('id_bv') is not None)
-        video_id = av_id
+        if 'bangumi/play' in url:
```

Comment:
You can capture this in regex and do `if mobj.group('bangumi')`

Reply:
> You can capture this in regex and do `if mobj.group('bangumi')`

ok